draft-terbu-sd-jwt-vc.md

```diff
@@ -31,8 +31,9 @@ organization="Authlete Inc. "
 .# Abstract
 
 This specification describes data formats as well as validation and processing
-rules to express Verifiable Credentials with JSON payload based on the SD-JWT
-format [@!I-D.ietf-oauth-selective-disclosure-jwt].
+rules to express Verifiable Credentials based on the Selective Disclosure
```

Comment:
why do we even need to change this?

Reply:
while this might be technically correct, I agree it might be confusing to first time readers. Either reverse to:
"based on the SD-JWT (Selective Disclosure for JWT and JWS with JSON payloads) format"
or leave it as is, as the full title is gehind the reference

Reply:
```suggestion
rules to express Verifiable Credentials with JSON payloads based on the Selective Disclosure
```

Reply:
because I think acronyms should be expanded when used the first time. we are deep in the details, but many do not know what SD-JWT is when they see it for the first time. and we should not assume that anyone reading this already knows what SD-JWT is.

Reply:
I guess this should be aligned with: https://github.com/vcstuff/draft-terbu-sd-jwt-vc/pull/132

Reply:
let's bikeshed the title in this PR for what is in scope for this PR. and then see if it is aligned or not with #132.

Reply:
I believe it is important to clarify what the specification is trying to achieve.
JWS: compact or JSON serialised JWS -> protection mechanism + serialisation
SD-JWT: SD-JWT serialisation (JWS+disclosures+tilde) or JSON serialised JWS -> protection mechanism + serialisation --> note that this could be fixed if the SD authors agree to. Issue is still open, a viable proposal is on the table.
JWT: compact serialised JWS -> claim set + protection
id token -> JWT + rules for the claims -> claim set + verification rules + protection

If I understand correctly, this specification defines a profile for Verifiable Credentials with the JWT claims set. So
vc+jwt -> claim set (VC + JWT claims), verification rules + protection (JWS or SD-JWT)

If my understanding is correct and if the specification is defining a new type, like in #132: vc+jwt, the specification can be defined as:
Verifiable Credential with JWT claims and protected by JWS (any serialisation) and/or SD-JWT

Or define a new brand:
- Verifiable Web Tokens (I prefer this one)
- JSON Verifiable Credentials
- JSON Web Credentials
- Verifiable Credential Tokens (don't like this one)

I agree with @selfissued  that "with JSON" payloads is clumsy/confusing.

Reply:
I don't think things to be that complicated...

Bottomline, there is a need for a credential format that uses JWT claim set in the payload, and depending on the use-case there may or may not be a need for selective disclosure. To achieve that we are exploring two approaches:
1. define a credential format without SD based on JWT RFC (JWT VC) and with SD based on the IETF SD-JWT draft (SD-JWT VC) - PR #120
2. define a credential format based on IETF SD-JWT that can be with or without SD (SD-WT VC) - PR #131

that's it.

Reply:
from what I have observed, majority seems to be favorable towards the approach in PR #131, because it gives one credential format and not two (consolidation, yay!), with one major discussion point being a `~` after a JWT VC when it is issued. 

a bit of the context... this trailing `~` is the result of the decision being taken in SD-JWT spec to better align the issuance and presentation formats, ([this PR](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/pull/293) does it, and it has not been merged yet). One option is to keep issuance and presentation formats separate in SD-JWT (as-is) so that there is no `~` when (SD-)JWT VC is issued. but that is not that clean from a pure SD-JWT processing perspective is I think the opinion of its authors (@danielfett, @bc-pi). 

so the question is really what to do with a trailing `~` when there is no SD in SD-JWT based VC.

Reply:
Hi @Sakurann . I agree that id doesn't have to be that complicated. IMO 2nd approach is cleaner as you can clearly split the data model and protection mechanisms.
See:
> If I understand correctly, this specification defines a profile for Verifiable Credentials with the JWT claims set. So
> vc+jwt -> claim set (VC + JWT claims), verification rules + protection (JWS or SD-JWT)

From your answer I understand you agree with "his specification defines a profile for Verifiable Credentials with the JWT claims set"
From design perspective, proposal 2 is better. Having a data model that depends on the signature format has limited re-usability. Let's assume a new signature or SD for JWT format is introduced. Would this required new data model specifications?